### PR TITLE
Validate subpuass input attachments are not bound to pipeline

### PR DESCRIPTION
**Pull request by locke-lunarg**
```
VUID-vkCmdDraw-None-02686
VUID-vkCmdDrawIndexed-None-02686
VUID-vkCmdDrawIndirect-None-02686
VUID-vkCmdDrawIndirectCount-None-02686
VUID-vkCmdDrawIndexedIndirect-None-02686
VUID-vkCmdDrawIndexedIndirectCount-None-02686
VUID-vkCmdDrawIndirectByteCountEXT-None-02686
VUID-vkCmdDrawMeshTasksNV-None-02686
VUID-vkCmdDrawMeshTasksIndirectNV-None-02686
VUID-vkCmdDrawMeshTasksIndirectCountNV-None-02686
: Every input attachment used by the current subpass must be bound to the pipeline via a descriptor set.
```

This case has two situations.
1) input attachment between subpass and fragment shader mismatch.
2) input attachment between fragment shader and descriptor set mismatch.

The second has been done in createGraphicsPipeline. Some UNASSIGNED validations are related to it. Changed those UNASSIGNED to 
`
VUID-VkGraphicsPipelineCreateInfo-layout-00756 
: layout must be consistent with all shaders specified in pStages
`

### CODE REVIEW

**Comment by mark-lunarg** [comment]
@locke-lunarg, is this for an existing issue?  Might see if it touches on #55 or #1568 at all...

**Comment by locke-lunarg** [comment]
> @locke-lunarg, is this for an existing issue (maybe #135)? Might see if it touches on #55 or #1568 at all...

This PR is for #135 , but not related to #55 and #1568

**Comment by locke-lunarg** [comment]
@mark-lunarg 

Should we change UNASSIGNED-CoreValidation-Shader-MissingDescriptor into 

VUID-VkGraphicsPipelineCreateInfo-layout-00756
VUID-VkComputePipelineCreateInfo-layout-00703
VUID-VkRayTracingPipelineCreateInfoNV-layout-03427
VUID-VkRayTracingPipelineCreateInfoKHR-layout-03427
: layout must be consistent with all shaders specified in pStages.

?

File: layers/core_validation_error_enums.h [hunk 1]
```diff
@@ -169,16 +169,13 @@ static const char DECORATE_UNUSED *kVUID_Core_DrawState_InvalidImageView = "UNAS
 //static const char DECORATE_UNUSED *kVUID_Core_DrawState_ViewportNotBound = "UNASSIGNED-CoreValidation-DrawState-ViewportNotBound";
 //static const char DECORATE_UNUSED *kVUID_Core_DrawState_VtxIndexAlignmentError = "UNASSIGNED-CoreValidation-DrawState-VtxIndexAlignmentError";
 
-static const char DECORATE_UNUSED *kVUID_Core_Shader_DescriptorNotAccessibleFromStage = "UNASSIGNED-CoreValidation-Shader-DescriptorNotAccessibleFromStage";
```
Comment (mark-lunarg) [inline layers/core_validation_error_enums.h:172]: @locke-lunarg, we typically leave these in here forever.  You may comment them out if you like, but it gives us traceablility back through the git history.

**Pull request merged**